review-service/model/redis-repository.js

```diff
@@ -0,0 +1,49 @@
+import { createClient } from 'redis'
+import { FIELDS } from '../controller/review-controller.js'
+import 'dotenv/config'
+
+const REDIS_URI = process.env.REDIS_URI || 'localhost:6379'
+console.log('(REDIS) Connecting to url:', REDIS_URI)
+const client = createClient({
+  url: `redis://default:@${REDIS_URI}`,
+})
+client.on('error', (err) => console.error('Redis client connection error', err))
+client.on('connect', () => console.log('REDIS CONNECTED'))
+await client.connect()
+
+const MINUTE = 60
+const HOUR = MINUTE * 60
+
+// for new writes and updates
+export const writeScoresToCache = async (newScores) => {
+  const cacheResults = await client.get(newScores.userId)
+  const cacheData = JSON.parse(cacheResults)
+  const data = {
+    userId: cacheData.userId,
+    totalReviews: (cacheData ? cacheData.totalReviews : 0) + 1,
```

Comment:
Similar bug, but perhaps `totalReviews` should just be set to `newScores.totalReviews`?

For instance, when a user's cached data in redis expires and is cleared, cacheData will be null, and this line will set `totalReviews` to 1. However, `newScores.totalReviews` will have the true number of reviews the user has from the DB, which could be like 3 or something